server/ladders.ts

```diff
@@ -465,7 +465,7 @@ class Ladder extends LadderStore {
 		// users must be different
 		if (user1 === user2) return false;
 
-		if (Config.fakeladder) {
+		if (Config.noipchecks) {
```

Comment:
Hm I wonder if `noipchecks` should necessary disable the rest of matchmaking? The way this is implemented, users can get matched with each other twice in a row, as well as get matched with people with wildly different Elo ratings. poke-env is probably happy with this, but it should probably be documented?